examples/acados_python/pendulum_on_cart/ocp/ocp_example_cost_formulations.py

```diff
@@ -199,6 +200,43 @@ def formulate_ocp(cost_version: str) -> AcadosOcp:
         ocp.model.cost_y_expr = ca.vertcat(x, u)
         ocp.model.cost_y_expr_e = x
         ocp.translate_cost_to_external_cost(cost_hessian='GAUSS_NEWTON')
+    elif cost_version == 'NLS_TO_EXTERNAL_P_GLOBAL':
+        ocp.cost.cost_type = 'NONLINEAR_LS'
+        ocp.cost.cost_type_e = 'NONLINEAR_LS'
+
+        p_global =struct_symSX([
```

Comment:
Not sure if using `struct_symSX` makes this example easier here. But no strong objection.